arch/arm/soc/atmel_sam/sam4s/soc.c

```diff
@@ -148,12 +110,13 @@ static ALWAYS_INLINE void clock_init(void)
 	 * register, should normally be the case.
 	 */
 	PMC->CKGR_MOR = CKGR_MOR_KEY_PASSWD
```

Comment:
Add comment above: /* Set the main fast RC oscillator to 12 MHz */

Also comment on line 105 should better start with "Setup"